src/Template/Bake/Template/view.ctp

```diff
@@ -148,11 +162,10 @@ foreach ($relations as $alias => $details):
             <%- foreach ($details['fields'] as $field): %>
                 <td><?= h($<%= $otherSingularVar %>-><%= $field %>) ?></td>
             <%- endforeach; %>
-            <%- $otherPk = "\${$otherSingularVar}->{$details['primaryKey'][0]}"; %>
                 <td class="actions">
-                    <?= $this->Html->link(__('View'), ['controller' => '<%= $details['controller'] %>', 'action' => 'view', <%= $otherPk %>]) %>
-                    <?= $this->Html->link(__('Edit'), ['controller' => '<%= $details['controller'] %>', 'action' => 'edit', <%= $otherPk %>]) %>
-                    <?= $this->Form->postLink(__('Delete'), ['controller' => '<%= $details['controller'] %>', 'action' => 'delete', <%= $otherPk %>], ['confirm' => __('Are you sure you want to delete # {0}?', <%= $otherPk %>)]) %>
+                    <?= $this->Html->link(__('View'), ['controller' => '<%= $details['controller'] %>', 'action' => 'view', <%= $relationsPrimaryKeys %>]) %>
+                    <?= $this->Html->link(__('Edit'), ['controller' => '<%= $details['controller'] %>', 'action' => 'edit', <%= $relationsPrimaryKeys %>]) %>
+                    <?= $this->Form->postLink(__('Delete'), ['controller' => '<%= $details['controller'] %>', 'action' => 'delete', <%= $relationsPrimaryKeys %>], ['confirm' => __('Are you sure you want to delete # {0}?', join([<%= $relationsPrimaryKeys %>], ' / '))]) %>
```

Comment:
The closing tag should be `?>`. This seems to already be wrong in the original templates etc.